srs/test/resources/output/encounterBundle.json

```diff
@@ -0,0 +1,208 @@
+{
+  "resourceType": "Bundle",
```

Comment:
This file should not be checked in. Please see the comment above about OUTPUT_PATH.